code/modules/mob/mob.dm

```diff
@@ -1167,6 +1167,8 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 	if((usr in GLOB.respawnable_list) && (stat == DEAD || isobserver(usr)))
 		var/list/creatures = list("Mouse")
 		for(var/mob/living/simple_animal/L in GLOB.alive_mob_list)
+			if(!(is_station_level(L.z) || is_admin_level(H.z))) // Prevents players from spawning in space
```

Comment:
minor spelling mistake, probably meant that H to be an L

Reply:
I forgot my second commit, whoops